src/resolver.js

```diff
@@ -58,6 +54,10 @@ function resolverFactory(target, options = {}) {
     findOptions.logging = findOptions.logging || context.logging;
     findOptions.graphqlContext = context;
 
+    _.each(contextToOptions, (as, key) => {
```

Comment:
_.assign

Reply:
Can't use assign as aliasing is supported